src/sources/il.json

```diff
@@ -1,17 +1,17 @@
 [
     {
-        "url": "http://www.svivaaqm.net/",
+        "url": "https://api.svivaaqm.net/v1/envista/",
         "adapter": "envista",
-        "name": "Israel",
+        "name": "Israel Envista",
```

Comment:
let change this so it follows the "key" style (sorry) so something like `israel-envista`